src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -133,7 +133,7 @@ def app_bundle(
 
 
 @app.command("run", requires_connection=True)
-@with_project_definition("native_app")
+@with_project_definition()
```

Comment:
Do we still get the behaviour that the command exits with a fatal error if the `native_app` section doesn't exist?

Reply:
good point - I will revert this logic, but keep this no-arg option available for future commands that might need it.